File: python/kode program/person.py
from abc import ABC, abstractmethod

class Person(ABC):
    def __init__(self, nama: str, umur: int):
        self.nama = nama
        self.umur = umur

    def set_nama(self, nama: str):
        self.nama = nama

    def get_nama(self) -> str:
        return self.nama

    def set_umur(self, umur: int):
        self.umur = umur

    def get_umur(self) -> int:
        return self.umur

    @abstractmethod
    def cetak_info(self):
        pass

File: java/kode program/Person.java
public abstract class Person {
    protected String nama;
    protected int umur;

    public Person(String nama, int umur) {
        this.nama = nama;
        this.umur = umur;
    }

    public void setNama(String nama) { this.nama = nama; }
    public String getNama() { return nama; }

    public void setUmur(int umur) { this.umur = umur; }
    public int getUmur() { return umur; }

    public abstract void cetakInfo();
}
